plugin/keyring_vault/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+CHECK_CXX_COMPILER_FLAG(-std=c++11 HAVE_STDCXX11)
```

Comment:
Changed this to check if memset_s function exists. This is cleaner as this is all what I cared about from C++11.